.pre-commit-config.yaml

```diff
@@ -1619,6 +1619,21 @@ repos:
         types: [python]
         files: ^providers/.*/src/airflow/providers/.*\.py$
         require_serial: true
+      - id: check-airflow-version-checks-in-core
+        language: pygrep
+        name: No AIRFLOW_V_3* imports in airflow-core
+        entry: "import AIRFLOW_V_3"
```

Comment:
```suggestion
        name: No AIRFLOW_V_* imports in airflow-core
        entry: "import AIRFLOW_V_"
```
No need for `_V_2` either? (not that I think we have any)

This'll need the svg etc regenerating